Bot.js

```diff
@@ -4,59 +4,73 @@ const client = new Discord.Client();
 
 //Constant for commands file
 const {commands} = require('./src/');
```

Comment:
What does it being in '{}' do to change it?

Reply:
So this way instead of a commands object there is a subReddit and bulkDelete object?